src/tests/dictionaryConcepts/components/CreateConceptForm.test.jsx

```diff
@@ -110,4 +110,36 @@ describe('Test suite for CreateConceptForm', () => {
     expect(wrapper.find('.form-group.answer')).toHaveLength(1);
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('should show all the possible options when creating another kind of conept', () => {
+    const props = {
+      state: {
+        id: '1',
+      },
+      concept: '',
+      addDescription: jest.fn(),
+      handleNewName: jest.fn(),
+      path: '',
+      toggleUUID: jest.fn(),
+      handleChange: jest.fn(),
+      handleSubmit: jest.fn(),
+      editable: false,
+      nameRows: [],
+      description: [],
+      addAnswer: jest.fn(),
+      answer: [],
+      disableButton: false,
+      allSources: [mockSource],
+      mappings: [{
+        id: 'uniqueid',
+        retired: false,
+      }],
+    };
+    const wrapper = mount(<CreateConceptForm {...props} />);
+    expect(wrapper.find('CreateConceptTable').length).toBe(1);
+    expect(wrapper.find('DescriptionTable').length).toBe(1);
+    expect(wrapper.find('AnswersTable').length).toBe(2);
```

Comment:
Kindly note that the sets and answers rows use the same components, hence there should be 2 instances of the AnswersTable component rendered.